handlers/handlers.go

```diff
@@ -200,20 +200,18 @@ type tileCoord struct {
 	x, y uint64
 }
 
-// extractTileCoord extracts and returns tile coordinates and an optional
-// extension from a byte slice and returns them. In case the byte slice is too
-// short or the number parsing cannot be performed, the returned error is
-// non-nil.  pcs is expected to be in order (z, y, x) when the parameter swapXY
-// is false, and in order (z, x, y) when swapXY is true. The extension is
-// extracted from the third element of pcs. Any elements in pcs beyond the
-// third are ignored.
-func extractTileCoord(pcs []string, swapXY bool) (tc tileCoord, ext string, err error) {
-	if len(pcs) < 3 {
-		err = fmt.Errorf("not enough elements to extract tile coordinates")
-		return
-	}
+// tileCoordFromString parses and returns tileCoord coordinates and an optional
+// extension from a the three parameters. The parameter z is interpreted as the
+// web mercator zoom level, it's supposed to be an unsigned integer that will
+// fit into 8 bit. The parameters x and y are interpreted as longitudinal and
+// lateral tileCoord indices for that zoom level, both are supposed be in the
+// range [0,2^z[. Additionally, y may also have an optional filename extension
```

Comment:
should this be `range [0, 2^z]`?

Reply:
I meant to exclude 2^z from the interval, but I failed to get the notation right. Changed to `[0,2^z)`.